docs/CODEOWNERS

```diff
@@ -63,10 +61,4 @@
 /frame/staking/ @paritytech/staking-core
 /frame/nomination-pools/ @paritytech/staking-core
 /frame/elections-phragmen/ @paritytech/staking-core
```

Comment:
Could you maybe sort them? Then it is easier to spot missing lie `election-provider-multi-phase`.